### ISSUE TITLE
Figure out how to report infeasibility

### ISSUE THREAD

**Issue by MuellMark** [MuellMark/Course-Scheduler#162]
Right now, there is no real way to know if a given set of requirements is infeasible other than the code crashing. I want to export to the file that it was infeasible.

This shouldn't be too hard to do, I'll just need to make a few test files to make sure it works right

**Comment by MuellMark** [comment]
I've added the first try except, but I think I need to make edits to the print statements, or something else needs to be edited, because just surrounding them in the block isn't working. 

I also need to take into account that there are different ways it could fail. either simplex itself could fail, or simplex works, and then the integer command. I need to make some cases and document what fails when, and also see what happens without the simplex commands, as it still passes through without issue for some reason. I'll think on it and come back to this after I eat dinner

**Comment by MuellMark** [comment]
I'm going to continue working on this now

**Comment by MuellMark** [comment]
I'm taking notes of how my different test files are breaking so I know where the try except statements need to be:

1. broke on lp.integer, has a forced time that isn't feasible
2. broke on lp.integer, has too many courses being 4 contact hours
3. broke on lp.integer, forced course at the same time as course it can;t be
4. broke on lp.integer, course with no feasible times

I'm going to go ahead and say that it breaks on lp.integer. That's nice to know, but that means I need to rewrite a bit of code. Right now, the simplex command will always run and create a solutino. If it fails on the integer step it needs to send a false or something back just to makes sure it doesn't print the messed up schedule to the csv. I'll look into how to do this correctly.

**Comment by MuellMark** [comment]
I can confirm that try except works, and it is now not doing anyhting in the export. I need to add some extra print/export statements and this will be complete.

**Comment by MuellMark** [comment]
At this point, the code doesn't print to the console if the schedule is infeasible, and it writes "infeasible" to the export csv. I've also tested this on all of the other test csv's to confirm that it hasn't broken.

This should be cleaned up, and from there I'll keep working on the list.

**Comment by MuellMark** [comment]
This is tied in with cleaning #52, but I removed all of the trouble shoot statements again and updated some of the documentation. Because the code was in a nice clean state and there are things that are now applicable to the website, I also decided to merge everything in. This issue is now closed. Tomorrow for my last hour I'll work on formatting the csv files that colby would be importing into my python code. Fixing the columns would be nice, but it's taking a back seat for more pressing issues
